sdk/test-utils/recorder-new/src/core-v1-recorder-node.ts

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { URLBuilder } from "@azure/core-http";
+import { DefaultHttpClient, WebResource, WebResourceLike } from "@azure/core-http";
+
+const paths = {
+  playback: "/playback",
+  record: "/record",
+  start: "/start",
+  stop: "/stop"
+};
+export class RecordingHttpClient extends DefaultHttpClient {
+  private static s_recordingServerUri = "http://localhost:5000";
+  private _httpClient: DefaultHttpClient;
+  private _recordingId?: string;
+  private _sessionFile: string;
+  private _startUri: string;
+  private _stopUri: string;
+  private _mode: string;
```

Comment:
Do we really need to have these all with underscore?

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/15826#discussion_r693310276
This is temporary to make sure everything is working, moving to a cleaner design soon.
Stay tuned. 😉